recipe/meta.yaml

```diff
@@ -28,22 +28,30 @@ build:
 
 requirements:
   build:
-    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('c') }}    # [not win]
     - {{ compiler('cxx') }}  # [not win]
-    - python 3.8  # [not win]
-    - ninja-base  # [not win]
-    - pkg-config  # [not win]
-    - patch     # [not win]
+    - python 3.8             # [not win]
+    - ninja-base             # [not win]
+    - pkg-config             # [not win]
+    - patch                  # [not win]
+    # libuv is a bootstrap dependency in js2c
+    # starting with 20.3 (https://github.com/nodejs/node/pull/46997)
+    - libuv
```

Comment:
any version requirements upstream?

Reply:
Generally, we do not put versions in `build`. However, it's not documented here https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/02_Recipes/Recipe_standards.md#the-requirements-section

Reply:
We should pin it to the same version as the one used in host. This helps with consistency, especially considering future rebuilds.